.github/workflows/deploy-storybook.yaml

```diff
@@ -41,4 +56,4 @@ jobs:
         checkout: false
         path: storybook/storybook-static
         install_command: echo Already done
-        build_command: pnpm run docs && pnpm build:storybook
+        build_command: pnpm run docs && pnpm build:storybook && cp openapi.json storybook/storybook-static/
```

Comment:
`cp openapi.json storybook/storybook-static/` is last as anything in `storybook/storybook-static` gets wiped in build step.